README.md

```diff
@@ -80,8 +82,18 @@ This extension provides 4 actions:
     ```php
     <?php
 
+    namespace app\controllers;
+
+    use Yii;
+    use yii\web\Controller;
+    use yii\helpers\Url;
+
+
     class SamlController extends Controller {
 
+        // Remove CSRF protection
+        public $enableCsrfValidation = false;
```

Comment:
Yes, we can just disable the ACS action, but I think there is no issue by disabling all.